src/main/drivers/bus_i2c.h

```diff
@@ -37,7 +37,23 @@ typedef enum I2CDevice {
     I2CDEV_COUNT
 } I2CDevice;
 
+// Maximum device instance count; consolidate with I2CDevice enum?
```

Comment:
IMO only `I2CDEV_COUNT` shall be used(and possibly pruned if necessary) , `I2CDEV_MAX` will only lead to problems